### PR TITLE
series-fixes

### PR DESCRIPTION
**Pull request by wissog**
AR-789

### CODE REVIEW

File: src/components/Sections/Lessons/tabs/Series/Container.js [hunk 1]
```diff
@@ -96,11 +96,11 @@ const SeriesContainer = ({ t }) => {
 
   const getHeader = node => node.id === "byTopics"
     ? t(`lessons.tabs.seriesTree.byTopics`)
-    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${node.name}`;
+    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${` `}${node.name}`;
```
Comment (gshilin) [inline src/components/Sections/Lessons/tabs/Series/Container.js:99]: it is just space, not ${` `}

File: src/components/Sections/Lessons/tabs/Series/Container.js [hunk 2]
```diff
@@ -96,11 +96,11 @@ const SeriesContainer = ({ t }) => {
...
 
   const renderKabbalist = node => {
     if (node?.children?.length > 0) {
-      const initLevel = 2;
+      const initLevel = 3;
```
Comment (gshilin) [inline src/components/Sections/Lessons/tabs/Series/Container.js:103]: Can you pls add a comment what does it mean?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Sections/Lessons/tabs/Series/Container.js (hunk 1 above)
Comment (wissog) [reply src/components/Sections/Lessons/tabs/Series/Container.js:99]: the space is needed. doesn't work other way

File: src/components/Sections/Lessons/tabs/Series/Container.js (hunk 2 above)
Comment (wissog) [reply src/components/Sections/Lessons/tabs/Series/Container.js:103]: it is used in the header in 2 rows

**Pull request merged**